repos/keg-components/src/utils/helpers/updateClassNames.js

```diff
@@ -0,0 +1,34 @@
+import { getPlatform } from 'KegGetPlatform'
+import { ensureClassArray } from '../../utils/helpers/ensureClassArray'
+const isWeb = getPlatform() === 'web'
+
+/**
+ * Helper to update the classNames on a Dom node
+ * @param {Object} element - Dom node element
+ * @param {Array[string]} classesRef - Array of classes currently on the element
```

Comment:
I think for maximum clarity, the type of `classesRef` should be 
`RefObject<Array<string>>` (replace `< and >` with `[ and ]` if you wish)

Reply:
yep that makes sense to me 👍